app/src/main/java/com/orange/ouds/app/ui/tokens/TokenPropertyIllustration.kt

```diff
@@ -126,4 +130,34 @@ fun SpacePaddingInlineWithImageIllustrationRow(
             contentScale = ContentScale.None
         )
     }
+}
+
+@Composable
+fun GridIllustrations() {
+    Image(
```

Comment:
Rendering is correct but images could be in a column to ensure this method will still work if we change the container. We could also factorize several properties:

```kotlin
@Composable
fun GridIllustrations() {
    Column(
        modifier = Modifier.padding(horizontal = OudsSpaceKeyToken.Fixed.Medium.value),
        verticalArrangement = Arrangement.spacedBy(OudsSpaceKeyToken.Fixed.Medium.value)
    ) {
        val resourceIds = listOf(
            R.drawable.il_tokens_grid_column_margin,
            R.drawable.il_tokens_grid_min_width,
            R.drawable.il_tokens_grid_max_width
        )
        resourceIds.forEach {
            Image(
                modifier = Modifier
                    .fillMaxWidth()
                    .background(OudsColorKeyToken.OnSurface.value), //TODO use BgEmphasizedPrimary token when available
                painter = painterResource(id = it),
                contentDescription = null
            )
        }
    }
}
```